nixos/modules/services/security/cfssl.nix

```diff
@@ -0,0 +1,205 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cfssl;
+in {
+  options.services.cfssl = {
+    enable = mkEnableOption ''
+      Whether to enable the CFSSL CA api-server.
```

Comment:
Only use `mkEnableOption "the CFSSL CA api-server"` here, `mkEnableOption` adds the "Whether to enable" thing.